### PR TITLE
[feature2] This is a new feature implemented in feature2 branch

### PR DESCRIPTION
**Pull request by mikegithuber**
These are the following changes.

- A new folder was created.

- A new file was created

### CODE REVIEW

**Comment by dhanushsurendra** [comment]
Good job mate!

File: feature2/another-feature.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was created in featue2 branch.
```
Comment (dhanushsurendra) [inline feature2/another-feature.txt:1]: Good job!

**Review by dhanushsurendra — APPROVED** [review]

File: feature2/another-feature.txt (hunk 1 above)
Comment (dhanushsurendra) [reply feature2/another-feature.txt:1]: Nice one

Comment (mikegithuber) [reply feature2/another-feature.txt:1]: Thank you!

**Pull request merged**